src/NetscriptFunctions/Extra.tsx

```diff
@@ -70,11 +74,14 @@ export function NetscriptExtra(): InternalAPI<INetscriptExtra> {
     iKnowWhatImDoing: (ctx) => () => {
       helpers.log(ctx, () => "Unlocking unsupported feature: window.tprintRaw");
       // @ts-expect-error window has no tprintRaw property defined
-      window.tprintRaw = Terminal.printRaw.bind(Terminal);
+      window.tprintRaw = (value: React.ReactNode) => {
+        Terminal.printRaw(<CustomBoundary key={`PlayerContent${customElementKey++}`} children={value} />);
+      };
     },
     printRaw: (ctx) => (value) => {
-      // Using this voids the warranty on your tail log
```

Comment:
Aww, I kind of liked the warning. XD Do we not need it anymore?

Reply:
The idea is that if we can provide adequate error checking / prevent react misuse from causing the actual game to crash, then it no longer needs to be an "advanced users only" feature with all sorts of warnings that it's unsupported functionality. Players could still have scripts that don't work if they don't use it right, but it isn't going to infect the rest of the game and send them to an error screen.